packages/core/__tests__/editor/nbsp.test.tsx

```diff
@@ -38,6 +39,19 @@ describe('Editor - NBSP', () => {
     expect(onContentChangeMock).toHaveBeenCalledTimes(1)
     expect(onContentChangeMock).toHaveBeenCalledWith(expectedOutput, '')
   })
+
+  it('Insert NBSP adds expected string', async () => {
+    cleanup = mockCreateRange()
+    const inputData = '<p>&#160;foo</p><p>bar</p>'
+    const expectedOutput = '<p>&#160;&#160;foo</p><p>bar</p>'
```

Comment:
Selitä tämä. Eli kun lisää yhden sitovan välilyönnin, niitä tulee kaksi?

Reply:
Tässä on yksi NBSP pohjalla ja lisätään toinen.  Tarkoitus on varmistaa ettei se alkuperäinen tipahda pois

Reply:
Okei, eli inputData on vähän kuin initialInputData ja siihen lisätään nbsp pikapainikkeella?

Reply:
just näin. Kaikki nää mun testit noudattaa sitä kaavaa